.changes/v3.9.0/965-features.md

```diff
@@ -0,0 +1,2 @@
+* **New Resource:** `vcd_rde_interface` to manage the Defined Interfaces required to use RDE types [GH-965]
+* **New Data Source:** `vcd_rde_interface` to fetch existing Defined Interfaces [GH-965]
```

Comment:
(This comment is not about change log, but an ask for the PR description.)

I suggest moving the "An example of a valid scenario" above the invalid example. Also, putting an inline comment into the invalid example that "it is wrong". This is to avoid accidental copy/pastes of the false example.